### ISSUE TITLE
Allow configuring empty value similarly to @CsvSource

### ISSUE THREAD

**Issue by mikemybytes** [mikemybytes/junit5-formatted-source#2]
JUnit 5 `@CsvSource` allows configuring empty value using a dedicated annotation parameter:
```java
public @interface CsvSource {
    // ...

    /**
    * The empty value to use when reading the {@linkplain #value records}.
    *
    * <p>This value replaces quoted empty strings read from the input.
    *
    * <p>Defaults to {@code ""}.
    *
    * @since 5.5
    */
    String emptyValue() default "";

    // ...
}
```

A similar approach could be adopted here.

**Comment by mikemybytes** [comment]
To be released with the next library version.

**Comment by mikemybytes** [comment]
Released in [0.2.0](https://github.com/mikemybytes/junit5-formatted-source/releases/tag/0.2.0).
